media-video/wireplumber/wireplumber-9999.ebuild

```diff
@@ -52,6 +52,7 @@ DEPEND="
 #		dev-lua/<NAME>[${LUA_USEDEP}]
 #	')
 RDEPEND="${DEPEND}
+	virtual/libintl
```

Comment:
It's needed in DEPEND too.

Reply:
Right, I was wondering where the PR would go wrong this time. Thank you for always having my back. ;)

Reply:
❤️